curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/-iterate-through-the-keys-of-an-object-with-a-for...in-statement.english.md

```diff
@@ -9,24 +9,40 @@ challengeType: 1
 Sometimes you may need to iterate through all the keys within an object. This requires a specific syntax in JavaScript called a <dfn>for...in</dfn> statement. For our <code>users</code> object, this could look like:
```

Comment:
```suggestion
Sometimes you may need to iterate through all the keys within an object. This requires a specific syntax in JavaScript called a <dfn>for...in</dfn> statement. For our <code>users</code> object, shown below, this could look like:
```
I just found myself wondering 'which `users` object?', since it's no longer on-screen on the right.

Reply:
We don't show one on the screen, because they will be different each time for each test (the online property value changes).  This paragraph and the code below it should look like:
```
Sometimes you may need to iterate through all the keys within an object. This requires a specific syntax in JavaScript called a <dfn>for...in</dfn> statement. In the example below, we have a <code>users</code> object and iterate through it to display the user's name to the console.

<blockquote>
const users = {<br>
&nbsp;&nbsp;Alan: {<br>
&nbsp;&nbsp;&nbsp;&nbsp;online: false<br>
&nbsp;&nbsp;},<br>
&nbsp;&nbsp;Jeff: {<br>
&nbsp;&nbsp;&nbsp;&nbsp;online: true<br>
&nbsp;&nbsp;},<br>
&nbsp;&nbsp;Sarah: {<br>
&nbsp;&nbsp;&nbsp;&nbsp;online: false<br>
&nbsp;&nbsp;}<br>
&nbsp;&nbsp;Ryan: {<br>
&nbsp;&nbsp;&nbsp;&nbsp;online: false<br>
&nbsp;&nbsp;}<br>
}
for (let user in users) {<br>
&nbsp;&nbsp;console.log(user);<br>
}<br><br>
// logs:<br>Alan<br>Jeff<br>Sarah<br>Ryan
</blockquote>
```

Reply:
Sure.  Are you thinking of reorganizing it like this?  I like this way around more, so the camper gets a clear logical flow: definition -> example object -> example use of definition

Reply:
Fixed this in latest push.